highbury/assetlist.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "highbury",
+  "assets": [
+    {
+      "description": "The native staking and governance token of the Blackfury ecosystem",
+      "denom_units": [
+        {
+          "denom": "ufury",
+          "exponent": 0
+        },
+        {
+          "denom": "fury",
```

Comment:
What's the relation to fanfury?

Reply:
Fanfury was originally launched on Terra Classic shortly before the UST depeg, Later the asset was migrated to the Juno network, where there were some internal issues that required a native chain for it to thrive.
Highbury network, will now be the home to the Fanfury project and FURY will be its native asset.

Reply:
Will the Juno variant be decommissioned? Is there any way to exchange juno.FURY to highbury.FURY , or maybe some airdrop?